### ISSUE TITLE
amp-video-docking + preroll issue

### ISSUE THREAD

**Issue by dgrbavcic** [ampproject/amphtml#26980]
amp-video-docking extension uses AD_START event to update controls, however it fails to do so on prerolls:

- Clicking play starts video play, preroll is started and AD_START is dispatched.
- User scrolls and player gets into the dock mode -- video-docking extension sets listeners at this time.
- Because AD_START was dispatched before, controls won't be updated.

Basically, dock extension won't update controls on ad started before player entered the dock mode at least once.
